plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -5260,6 +5272,17 @@ protected Answer execute(ModifyStoragePoolCommand cmd) {
         }
     }
 
+    private boolean getHardwareAcceleationSupportForDataStore(ManagedObjectReference host, String dataStoreName) throws Exception {
```

Comment:
```suggestion
    private boolean getHardwareAccelerationSupportForDataStore(ManagedObjectReference host, String dataStoreName) throws Exception {
```

typo correction - Acceleration

Reply:
Done